### PR TITLE
Add some UI feedback when the user requests the features for a repo.

### PR DESCRIPTION
**Pull request by jimCresswell**
Add some UI feedback when the user click the submit button. Addresses #63 .

### CODE REVIEW

File: views/index.hbs [hunk 1]
```diff
@@ -5,7 +5,8 @@
     <h2>Get features from a project</h2>
     <form action="/" method="get">
       <label>Project Git repo URL (https only please): <input type="text" name="repo_url" /></label>
-      <button class="call-to-action">Submit</button>
+      <button class="call-to-action" onclick="document.body.classList.add('waiting')">Submit</button>
```
Comment (jimCresswell) [inline views/index.hbs]: Dirty. But I don't want to get into client side JS just yet.

**Comment by sponte** [comment]
Awesome!

**Comment by sponte** [comment]
Was meaning to merge lol

**Comment by jimCresswell** [comment]
Oops :smiley: . inline javascript, it's so last decade.

**Comment by sponte** [comment]
It's fine as this is not a final solution

**Comment by jimCresswell** [comment]
I know, getting it working comes first.

**Pull request merged**